server.go

```diff
@@ -119,6 +120,15 @@ func (s *Server) Parse(req *protocol.ParseRequest) *protocol.ParseResponse {
 		lang = GetLanguage(req.Filename, []byte(req.Content))
 	}
 
+	// If the code Content is empty, just return an empty reponse
+	if req.Content == "" {
+		logrus.Info("Empty code received, returning empty UAST")
```

Comment:
Not very relevant information about the server operation itself, maybe a `Debug` is better?